devops/kubernetes/misc/origin-scripts.yaml

```diff
@@ -0,0 +1,9 @@
+kind: Pod
+apiVersion: v1
+metadata:
+  name: origin-scripts
+spec:
+  containers:
+    - name: origin-scripts
+      image: "gcr.io/origin-214503/dev/origin-cron:latest"
```

Comment:
Is this meant to be `origin-scripts`?

Reply:
Nope.  Didn't want to setup a build process for another image when the `origin-cron` image would do just fine for this purpose.